src/support/command/update-command.js

```diff
@@ -1,29 +1,48 @@
 const BaseCommand = require('./base-command')
 const { putSpec } = require('../../requests/spec')
+const {
```

Comment:
Could you use the same format as the other - on a single line with no trailing comma?